examples/terraform-examples/jfrog-platform-aws-install/README.md

```diff
@@ -0,0 +1,70 @@
+# JFrog Platform Installation in AWS with Terraform
+This example will prepare the AWS infrastructure and services required to run Artifactory and Xray (installed with the [jfrog-platform Helm Chart](https://github.com/jfrog/charts/tree/master/stable/jfrog-platform)) using Terraform:
+1. The AWS VPC
+2. RDS (PostgreSQL) as the database for each application
+2. S3 as the Artifactory object storage
+3. EKS as the Kubernetes cluster for running Artifactory and Xray with pre-defined node groups for the different services
+
+The resources are split between individual files for easy and clear separation.
+
+
+## Prepare the JFrog Platform Configurations
```

Comment:
Add a section to instruct of setting up the `aws` CLI before starting `terraform`. Just explain the need for a configured account with the right permissions.